src/main/kotlin/fi/metatavu/timebank/api/impl/TimeEntriesApi.kt

```diff
@@ -36,9 +36,11 @@ class TimeEntriesApi: TimeEntriesApi, AbstractApi() {
     }
 
     override suspend fun forecastTimeEntriesDeleteWebhook(forecastDeleteWebhookEvent: ForecastDeleteWebhookEvent, forecastDeleteWebhookKey: String?): Response {
-            if (forecastDeleteWebhookKey != forecastWebhookKey) {return createUnauthorized(message = "Invalid key!")}
-                timeEntryController.deleteEntry(forecastId = forecastDeleteWebhookEvent.`object`!!.id)
-                return createNoContent()
+        if (forecastDeleteWebhookKey != forecastWebhookKey) {return createUnauthorized(message = "Invalid key!")}
```

Comment:
One-liner statements do not need curly brackets. And with more webhooks planned in the near future, this authorization could propably be abstracted, similarly to who loggedUserId is being handled across this application.